src/docs/sdk/features.mdx

```diff
@@ -231,6 +231,58 @@ If Performance Monitoring is both supported by the SDK and enabled in the client
 - span status must match HTTP response status code ([see Span status to HTTP status code mapping](/sdk/event-payloads/span/))
 - when network error occurs, span status must be set to `internal_error`
 
+### HTTP Client errors
+
+HTTP Client errors are automatically captured and sent to `https://sentry.io`.
+
+The HTTP Client integration should have 3 configuration options:
+
+* `captureFailedRequests` defaults to `false` due to PII reasons.
+* `failedRequestStatusCodes` defaults to `500` to `599`, this configuration option accepts a `List` of `HttpStatusCodeRange` which is a range of HTTP status code -> `min` to `max` or a single `status_code`.
+* `failedRequestTargets` defaults to `.*`, this configuration option accepts a `List` of `String` that may be Regular expressions as well, similar to <Link to="/sdk/performance/#tracepropagationtargets">tracePropagationTargets</Link>.
+
+Then the HTTP Client integration should not capture the error:
+
+* If `captureFailedRequests` is disabled.
+* If the HTTP Request url isn't a match for any of the `failedRequestsTargets`.
+* If the HTTP Response status code isn't within the defined ranges in `failedRequestStatusCodes`.
+
+The HTTP Client integration should capture error events with the following properties:
```

Comment:
`m`: What is the title and content of the error event? Is it the exception itself?

Reply:
That depends on the original or synthetic exception/stack trace, I will add a note about the title I've used.

Reply:
I don't see the note, or am I missing something?

Reply:
Its there, https://develop-git-chore-http-client-errors.sentry.dev/sdk/features/#http-client-errors
As a subitem of the stack trace interface, the title example.